datadog-oci-orm/metrics-setup/schema.yaml

```diff
@@ -52,7 +63,7 @@ variables:
   function_subnet_id:
     title: Function Subnet OCID
     type: oci:core:subnet:id
-    description: The OCID of the subnet to be used for the function app. Required if not creating the VCN.
+    description: The OCID of the subnet to be used for the function app. If create VCN is checked, then that will take precedence over the selection of the subnet.
```

Comment:
The subnet to be used by the function app that will be deployed. If "Create VCN" is checked, the new VCN will overwrite this selection.